### PR TITLE
Some additions if conda and R haven't been setup previously

### PR DESCRIPTION
**Pull request by ha0ye**
I ran into a few setup hiccups (I think related to not having setup conda and R before on HPG). Please look over these additions to the vignette to see if they do the trick.

I was worried about the version of `retriever` obtained through `reticulate::conda_install`, but it seems to be updated to point at the latest one. Does that sound correct, @henrykironde, @ethanwhite?

### CODE REVIEW

**Comment by henrykironde** [comment]
@ha0ye when you say 

> obtained through reticulate::conda_install, but it seems to be updated to point at the latest one.

Do you mean with the latest released version or with master?

**Comment by ha0ye** [comment]
I’m not sure. It looked like install_csv had 5 arguments instead of 4, which was the functionality that we depend on via rdataretriever.

**Comment by henrykironde** [comment]
@ha0ye, currently there are some problems with args for now. I am working to release all new tools by the end of the week. I am sure it will be resolved. I will update the master branch today and this may be a temp solution if you install both python and R packages from master tomorrow.

**Comment by ethanwhite** [comment]
> I was worried about the version of retriever obtained through reticulate::conda_install, but it seems to be updated to point at the latest one. Does that sound correct, @henrykironde, @ethanwhite?

This will install the current release which should have everything we need. If I'm reading correctly everything is working right yes?

File: vignettes/hipergator-install.Rmd [hunk 1]
```diff
@@ -16,49 +16,63 @@ knitr::opts_chunk$set(
...
+4. Make sure that you have the specified folder for storing R packages (substituting your 
+  username for <USERNAME>):
 
-4. Load and open the R module (this loads R for use)
+```bash
+   mkdir /home/<USERNAME>/R_libs
+```
 
-    ```bash
+5. Load and open the R module (this loads R for use)
```
Comment (ethanwhite) [inline vignettes/hipergator-install.Rmd]: I would say "Load the R module (which makes R available) and open R"

**Review by ethanwhite — APPROVED** [review]
One minor wording suggestion. Other than that LGTM!

**Pull request merged**